Source/Core/Core/FifoPlayer/FifoDataFile.h

```diff
@@ -18,11 +18,11 @@ class IOFile;
 
 struct MemoryUpdate
 {
-  enum Type
+  enum class Type
```

Comment:
Given that this has to fit into an u8 for serialization, can you mark this as such? `enum class Type : u8`